modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/footer/index.html

```diff
@@ -0,0 +1,20 @@
+<div class="cp_footer_home">
+	<div class="cp_footer_container">
+	<h6 class="cp_footer_text" data-lfr-editable-id="cp_footer_text" data-lfr-editable-type="rich-text">
+			�� 2023 Liferay Inc. ${languageUtil.get(locale,
+			"all-rights-reserved")} &nbsp; &nbsp; &nbsp; &nbsp; |
+		</h6>
+
+		<h6 class="cp_footer_text">
+		<a href="/web/cp-footer/cookie-policy" data-lfr-editable-id="link" data-lfr-editable-type="link">
```

Comment:
`<a href` should be indented since its nested inside the `<h6` above it

Reply:
The indentation is being generated automatically by the formatting commands.